src/Storage/Storage.Management/help/Enable-AzStorageContainerDeleteRetentionPolicy.md

```diff
@@ -0,0 +1,203 @@
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/az.storage/enable-azstoragecontainerdeleteretentionpolicy
+schema: 2.0.0
+---
+
+# Enable-AzStorageContainerDeleteRetentionPolicy
+
+## SYNOPSIS
+Enable delete retention policy for Azure Storage blob contatiners.
+
+## SYNTAX
+
+### AccountName (Default)
+```
+Enable-AzStorageContainerDeleteRetentionPolicy [-ResourceGroupName] <String> [-StorageAccountName] <String>
+ -RetentionDays <Int32> [-PassThru] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### AccountObject
+```
+Enable-AzStorageContainerDeleteRetentionPolicy -StorageAccount <PSStorageAccount> -RetentionDays <Int32>
+ [-PassThru] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### BlobServicePropertiesResourceId
+```
+Enable-AzStorageContainerDeleteRetentionPolicy [-ResourceId] <String> -RetentionDays <Int32> [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Enable-AzStorageContainerDeleteRetentionPolicy** cmdlet enables delete retention policy for Azure Storage blob Contatiner.
+
+## EXAMPLES
+
+### Example 1: Enable delete retention policy for Blob containers
+```
+C:\PS>Enable-AzStorageContainerDeleteRetentionPolicy -ResourceGroupName "myresourcegroup" -AccountName "mystorageaccount" -PassThru -RetentionDays 3
+
+Enabled Days
+------- ----
+   True    3
+```
+
+This command enables delete retention policy for Blob Contatiners, and set deleted blob retention days to 3.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+Display ServiceProperties
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Resource Group Name.
+
+```yaml
+Type: System.String
+Parameter Sets: AccountName
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceId
+Input a Storage account Resource Id, or a Blob service properties Resource Id.
+
+```yaml
+Type: System.String
+Parameter Sets: BlobServicePropertiesResourceId
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -RetentionDays
```

Comment:
I have static analysis check improved recently. It seems that plural parameter name will cause a signature error. I am sure whether the error will be report when you try to merge the codes into master. Since the plural name here is caused by legacy codes, you can suppress the issue if the issue is reported when merge back to master.